server/src/test/java/org/elasticsearch/search/query/QueryStringIT.java

```diff
@@ -295,6 +295,35 @@ public void testLimitOnExpandedFields() throws Exception {
                         + (CLUSTER_MAX_CLAUSE_COUNT + 1)));
     }
 
+    // The only expectation for this test is to not throw exception
+    public void testLimitOnExpandedFieldsButIgnoreUnmappedFields() throws Exception {
+        XContentBuilder builder = jsonBuilder();
+        builder.startObject();
+        builder.startObject("type1");
+        builder.startObject("properties");
+        for (int i = 0; i < CLUSTER_MAX_CLAUSE_COUNT; i++) {
+            builder.startObject("field" + i).field("type", "text").endObject();
+        }
+        builder.endObject(); // properties
+        builder.endObject(); // type1
+        builder.endObject();
+
+        assertAcked(prepareCreate("ignoreunmappedfields").addMapping("type1", builder));
```

Comment:
@zacharymorn I merged master in your pr and now there's a test compilation error. Can you take a look ?

Reply:
Looks like there’s change to `CreateIndexRequestBuilder.addMapping` method . I just updated the test and pushed up the commit. It should work for CI now.

Reply:
I merged master in and looks like there’s change in  `CreateIndexRequestBuilder.addMapping` . Just updated the test and pushed up the commits.  The CI is still failing though it was broken for an seemingly unrelated integration test `org.elasticsearch.snapshots.DedicatedClusterSnapshotRestoreIT.testMasterShutdownDuringSnapshot`. Is this something I should be looking into as well ?